articles/libraries/machine_learning/concepts/Qccc.Training.md

```diff
@@ -0,0 +1,56 @@
+
+
+# Classifier Training Process and Results
+
+Training of a circuit-centric quantum classifier is a process with many moving parts that require the same (or slightly larger) amount of
+calibration by trial and error as training of traditional classifiers. Here we define the main concepts and ingredients of this training process. 
+
+# Training/testing schedules
+The the context of classifier training a `schedule` describes a subset of data samples in an overall training or testing set. A schedule is usually defined as a
+collection of sample indices.
+
+# Parameter/bias scores
+Given a candidate parameter vector and a classifier bias, their `validation score` is measured relative to a chosen validation schedule S and is expressed by a number of misclassifications
+ counted over all the samples in the schedule S
+
+
+# Hyperparameters
+The model training process is governed by certain pre-set values called `hyperparameters`:
+
+## Learning rate (LR) 
+
+is one of key hyperparameters. It defines how much current stochastic gradient estimate impacts the parameter update. The size of parameter update delta is 
+proportional to LR. Smaller LR values lead to slower parameter evolution and slower convergence, but excessively large values of LR may break the convergence 
+altogether as the gradient descent never commits to a particular local minimum. While LR is adaptively adjusted by the training algorithm to some extent, s
+electing a good initial value for it is important. A usual default initial value for LR is 0.1
```

Comment:
```suggestion
electing a good initial value for it is important. A usual default initial value for LR is 0.1.
```
Can we suggest any insight as to why 0.1 is a good learning rate, or perhaps provide citations to that effect?

Reply:
As an example, learning rate and its common default values are discussed on page 86 or I. Goodfellow, Y. Bengio, A. Courville, Deep Learning, MIT Press, 2016, see also a good discussion at https://www.jeremyjordan.me/nn-learning-rate/

Reply:
I have inlined the reference to Deep Learning in one of the latest commits.